compiler/fir/raw-fir/psi2fir/tests/org/jetbrains/kotlin/fir/builder/RawFirBuilderTotalKotlinTestCase.kt

```diff
@@ -244,9 +244,10 @@ class RawFirBuilderTotalKotlinTestCase : AbstractRawFirBuilderTestCase() {
                         it is KtQualifiedExpression ||
                         it is KtPropertyDelegate ||
                         it is KtConstructorCalleeExpression && it.parent is KtAnnotationEntry ||
+                        it is KtConstructorCalleeExpression && it.parent is KtSuperTypeCallEntry ||
```

Comment:
Nit: this could be combined with the above? (I don't have strong preference, though.)

Also, am I right to read these lines: `KtConstructorCalleeExpression` as annotation entry or being used in delegated `super` call is allowed?

Reply:
Yeah, specifically "being allowed" should mean "it's ok that these are not traversable through FIR". Let me add a local variable with a descriptive name.